src/Juvix/Compiler/Backend/Geb/Translation/FromCore.hs

```diff
@@ -322,7 +380,14 @@ fromCore tab = case tab ^. Core.infoMain of
     convertCase :: Core.Case -> Trans Morphism
     convertCase Core.Case {..} = do
       if
-          | null branches -> MorphismAbsurd <$> convertNode _caseValue
+          | null branches -> do
+              x <- convertNode _caseValue
+              return $
+                MorphismAbsurd
+                  Absurd
+                    { _absurdType = codomainType,
```

Comment:
So why didn't you fix this? This will always give an error because `branches` is empty.

Reply:
`_absurdType = convertType (Info.getInfoType _caseInfo)`

Reply:
Wait, I did. It's very strange.